storagedriver/azure/azure.go

```diff
@@ -40,28 +40,28 @@ func init() {
 
 type azureDriverFactory struct{}
 
-func (factory *azureDriverFactory) Create(parameters map[string]string) (storagedriver.StorageDriver, error) {
+func (factory *azureDriverFactory) Create(parameters map[string]interface{}) (storagedriver.StorageDriver, error) {
 	return FromParameters(parameters)
 }
 
 // FromParameters constructs a new Driver with a given parameters map.
-func FromParameters(parameters map[string]string) (*Driver, error) {
+func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 	accountName, ok := parameters[paramAccountName]
-	if !ok {
+	if !ok || accountName.(string) == "" {
```

Comment:
@stevvooe fixed this one (and the ones below). however, I noticed the same goes with the line here: https://github.com/docker/distribution/blob/03406ab708a284dd27e64f2157784b65943d882d/storagedriver/s3/s3.go#L89 that should probably be fixed.